radar/languages_frameworks/swagger.yml

```diff
@@ -0,0 +1,13 @@
+version: 1
```

Comment:
Already exists in the radar under `Open API (Swagger)`
Remove this one and revise existing description if necessary instead.

Reply:
removed